core/storage/user/gae_models.py

```diff
@@ -200,8 +200,10 @@ def export_data(user_id):
             'subject_interests': user.subject_interests,
             'first_contribution_msec': user.first_contribution_msec,
             'preferred_language_codes': user.preferred_language_codes,
-            'preferred_site_language_code': user.preferred_site_language_code,
-            'preferred_audio_language_code': user.preferred_audio_language_code
+            'preferred_site_language_code':
```

Comment:
Why did these lines need to be changed? (See the diff in "Files Changed".)

In general, try to avoid unnecessary changes in your PRs.

Reply:
These lines had to be changed since the linter was complaining about them.

Reply:
Ah ok -- weird. Is this the CI linter or your local linter?

@Hudda -- could you please confirm that this is expected? If so, why didn't the linter complain before? Might want to look into that...

Reply:
@seanlip This is not expected. Actually, this is a false positive (The line length is less than 80). I will try to find the reason behind this behavior. Thanks for reporting this.

Reply:
Thank you, @Hudda! I suggest working with @varun-tandon directly since I think this might be relevant to his local setup. I'll introduce both of you by email.

Reply:
The linter no longer complains (see my email). I'll change this back.